js/build.js

```diff
@@ -117,8 +117,8 @@ Fliplet().then(function() {
                   vmData.sendValidationLabel = 'Continue';
                   vmData.disableButton = false;
                 })
-                .catch(function() {
-                  vmData.emailError = true;
+                .catch(function(error) {
+                  vmData.emailError = error.responseJSON.message;
```

Comment:
I think you can use `vmData.emailError = Fliplet.parseErrpr(error);`, @tonytlwu can you confirm?

Reply:
Yes, please use `Fliplet.parseError()` here

Reply:
@tonytlwu @squallstar 
Fixed.